src/derivatives/atlas.md

```diff
@@ -0,0 +1,615 @@
+      ├─ sub-01_space-MNI305_atlas-AAL_dseg.tsv
+      ├─ sub-01_space-MNI305_atlas-AAL_probseg.nii.gz
+      ├─ sub-01_space-MNI305_label-brain_mask.nii.gz
+      ├─ sub-01_space-MNI305_label-head_mask.nii.gz
+      ├─ sub-01_space-MNI305_T1w.nii.gz
+      ├─ sub-01_space-MNI305_T1w.json
+      ├─ sub-01_space-T1w_label-brain_mask.nii.gz
+      ├─ sub-01_space-T1w_label-head_mask.nii.gz
+      ├─ sub-01_space-T1w_T1w.nii.gz
+      └─ sub-01_space-T1w_T1w.json
+```
+
+## Multi-subject template and atlases and deriving an existing template/atlas
+
+Atlasing multiple individual brains is a higher-than-first-level analysis,
+as it requires first generating derivatives for the individuals (for example,
+a transformation to align them into a standardized space) and later aggregate
+and distill the sample-pooled knowledge and feature maps.
+Similarly, deriving from an existing template and atlases is also
+a higher-than-first-level analysis as it builds on a previous analysis.
+
+**Multi-subject templates and atlases**.
+While at the subject level analysis it is the individual brain that establishes
+stereotaxy.
+At higher-than-first-level stereotaxy is supported by templates, which are
+encoded through the [`tpl-<label>` entity](../glossary.md#template-entities).
+For the pipeline that generated the MNI152NLin2009cAsym, the outputs would look
+like the following example:
+
+```Text
+mni152nlin2009casym-pipeline/
+└─ tpl-MNI152NLin2009cAsym/
+   └─ anat/
+      ├─ tpl-MNI152NLin2009cAsym_res-1_label-brain_mask.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-1_label-eye_mask.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-1_label-face_mask.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-1_label-head_mask.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-1_label-CSF_probseg.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-1_label-GM_probseg.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-1_label-WM_probseg.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-1_T1w.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-1_T1w.json
+      ├─ tpl-MNI152NLin2009cAsym_res-2_label-brain_mask.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-2_label-eye_mask.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-2_label-face_mask.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-2_label-head_mask.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-2_label-CSF_probseg.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-2_label-GM_probseg.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-2_label-WM_probseg.nii.gz
+      ├─ tpl-MNI152NLin2009cAsym_res-2_T1w.nii.gz
+      └─ tpl-MNI152NLin2009cAsym_res-2_T1w.json
+```
+
+Since multi-subject templates and atlas involve the spatial normalization of
+subjects by means of image registration processes, it is RECOMMENDED to store
+the resulting transforms for each of the subjects employed to create the
+output.
+Please note that the specification for spatial transforms (BEP 014) is currently
+under development, and therefore, the specification of transforms files may
+change in the future.
```

Comment:
@PeerHerholz this is the mention.

I didn't want to explicitly get down the `from-` / `to-` issue because that has great potential to change (or establish some extra rules so that it is unambiguous).